.github/workflows/helm-lint.yml

```diff
@@ -0,0 +1,25 @@
+name: Lint the Helm chart
+
+on:
+  pull_request:
+    branches:
+    - main
```

Comment:
I think the PR trigger here needs to work on more branches than just `main` as we have PRs at times for prior releases.  See lines 10-13 in the deleted `.travis.yml` (lines 10, 11 and 13 may be left over from how we named things in ICP, but at least the `^release-[0-9]+\..*$/` pattern should be there.

Reply:
That's a good point. I'll just take off the branch specification entirely because we probably want the lint running on every PR.